deployment/terraform/efs.go

```diff
@@ -0,0 +1,87 @@
+package terraform
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+const (
+	efsDirectory  = "/mnt/mattermost-data"
+	svcScript     = "/opt/mount.sh"
+	efcServiceDir = "/lib/systemd/system/efs-mount.service"
+	efsService    = `[Unit]
+Description=Script to run after fstab to change ownership of mount directory
+After=local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/bin/bash -c "/opt/mount.sh"
+
+
+[Install]
+WantedBy=multi-user.target
+`
+)
+
+func (t *Terraform) setupEFS(extAgent *ssh.ExtAgent) error {
+	for _, val := range t.output.Instances {
+		sshc, err := extAgent.NewClient(val.PublicIP)
+		if err != nil {
+			return fmt.Errorf("error in getting ssh connection %w", err)
+		}
+		defer sshc.Close()
+
+		cmd := fmt.Sprintf("sudo mkdir -p %s", efsDirectory)
+		_, err = sshc.RunCommand(cmd)
+		if err != nil {
+			return fmt.Errorf("error creating the mounting point: %w", err)
+		}
+
+		mlog.Info("Updating /etc/fstab to mount EFS", mlog.String("fsid", t.output.EFSAccessPoint.FileSystemId))
+		// setting uid and gid to ubuntu user
+		fstabEntry := fmt.Sprintf("%s.efs.%s.amazonaws.com:/ %s nfs4 nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport,_netdev 0 0", t.output.EFSAccessPoint.FileSystemId, t.config.AWSRegion, efsDirectory)
```

Comment:
Nit: can we make this a multiline? Something like 
```suggestion
		fstabEntry := fmt.Sprintf("%s.efs.%s.amazonaws.com:/ %s nfs4 nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport,_netdev 0 0",
			t.output.EFSAccessPoint.FileSystemId,
			t.config.AWSRegion,
			efsDirectory)

```